.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
An example, I removed this file at all.

Reply:
Got it, we can close this PR. I am going to open a new PR.